pom.xml

```diff
@@ -1,85 +1,169 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- $Id$ -->
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
     <modelVersion>4.0.0</modelVersion>
+
     <groupId>org.mule.tools</groupId>
     <artifactId>mule-assembly-verifier</artifactId>
-    <version>1.5-SNAPSHOT</version>
+    <version>2.0.0-SNAPSHOT</version>
     <packaging>maven-plugin</packaging>
     <name>Mule Assembly Verifier Plugin</name>
 
+    <!-- //@todo[@theales]: What do you think about this new description? -->
     <description>
-        Verify contents of the output assembly against a controlled set of libraries.
+        Provide goals for performing important tasks in the Mule's distro assembly lifecycle like verify its contents or
```

Comment:
"Provide goals" is redundant, since every maven plugin provides goals

Reply:
Inspiration was taken from [here](https://mvnrepository.com/artifact/org.apache.maven.plugins) 👇  😅   😜

![image](https://user-images.githubusercontent.com/1432287/124649840-a0bd8880-de5e-11eb-982f-687b7b7aa0df.png)

Does it look better now?